src/vectormenu.js

```diff
@@ -15,129 +15,211 @@ limitations under the License.
 */
 
 const {app, shell, Menu} = require('electron');
+const { _td } = require('./language-helper');
 
```

Comment:
the changes in this file will cause inconsistency, as we don't drive all the menus.

E.g on macos the primary menu will always be based on the OS language and not Element language vs other menus which will be the other way around.

![image](https://user-images.githubusercontent.com/2403652/116092141-76509180-a69d-11eb-88ad-74af68a09560.png)


^ this menu is not defined by us

Reply:
I totally agree with you, but my change minise the area with this type of inconsistencies.

To give you a representation of the issue, I have created a translation file that interpolates all the strings to `a` 

![Screen Shot 2021-04-26 at 14 25 15](https://user-images.githubusercontent.com/769871/116092592-d8a99200-a69d-11eb-8d19-5efbd92aaadd.png)

The user would end up having multiple language in the same menu if they were to have a different language set on their OS and on Element.

Again there are exceptions like the one you pointed above but that seem like a lesser evil to me

Reply:
My suggestion is we should probably make the primary menu explicit so that it is consistent, it should just need a label of the same as the application name, e.g `Element` / `Element Nightly` and then it'll become the primary menu

Reply:
Turns out the primary menu was explicit. However when running the app using `yarn start` the `label` was a mismatch with the brand name, and it was missing the proper role of `appMenu`. 

I have changed that and it should be all good to go!